index.html

```diff
@@ -0,0 +1,229 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css?family=Poppins:100,100i,200,200i,300,300i,400,400i,500,500i,600,600i,700,700i,800,800i,900,900i&amp;display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="media.css">
+
+    <title>Mobile Portfolio</title>
+</head>
+<body>
+        <header>
+        <nav>
+        <nav class="Main-container"><p class="logo"><a="">My Logo</a></p>
+        <nav class="menu"><i class="fa fa-bars"></i></nav>
+        <nav class="menu2">
+            <ul>
+                <li>Portfolio</li>
+                <li>About</li>
+                <li>Contact</li>
+            </ul>
+        </nav>
+        </nav>
+        </nav>
+        </header>
+<section class="top">
+       <h1>I’m Abhishek
+        Glad to see you!</h1>
+    <p class ="About">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    <a href="#" class="Meet">LET'S CONNECT</a>
+    <div class="Links">
+    <ul>
+       
+       <li><img src="assets/images/Vector.svg" alt="icon1"></li>
+       
+       <li><img src="assets/images/Linkedin.svg" alt="icon2"></li>
+       <li><img src="assets/images/Vector (1).svg" alt="icon3"></li>
+       <li><img src="assets/images/Vector (2).svg" alt="icon4"></li>
+       <li><img src="assets/images/Vector (3).svg" alt="icon5"></li>
+        </ul>
+         </div>
+</section>
+
+<section class="works">
+    <div class="project1">
+        <div class="img1"><img src="assets/images/Tonic.png" alt="T"></div>
+        <div class="card1"><img src="assets/images/card1_ds.png" alt="T"></div>
+        <div class="task"><h2>Tonic</h2>
+            <div class="list">
+            <ul>
+                <li class="canopy">CANOPY</li> 
+                <li><img src="assets/images/Counter.png" alt="dot"></li>
+                <li>Backend Dev</li>
+                <li><img src="assets/images/Counter.png" alt="dot"></li>
+                <li>2015</li>
+            </ul>
+            </div>
```

Comment:
In this case, I think it'd have been better if you used <ul> element as the wrapper of the <li> elements  instead of <div> as this will make your code more efficient than spreading the code over several lines